### DISCUSSION TITLE
Use `mdBook` for the application documentation

### DISCUSSION THREAD

**Discussion by josecelano** [torrust/torrust-index-gui#186] [Proposals]
For now, we have the application documentation on some markdown files in the docs folder:

https://github.com/torrust/torrust-index-frontend/blob/develop/docs/index.md

But we are not publishing it anywhere.

For the [tracker](https://docs.rs/torrust-tracker/3.0.0-alpha.3/torrust_tracker/) and [backend](https://docs.rs/torrust-index-backend/2.0.0-alpha.3/torrust_index_backend/), we have the docs on https://docs.rs/.

Since we do not have any Rust crate for this repo, we could use a `mdBook`:

https://github.com/rust-lang/mdBook

and publish it to Github Pages.

The documentation for this repo could be extensible if we include here:

- User guide
- Deployment guides (maybe this one should go on the [Index](https://github.com/torrust/torrust-index) project)

cc @da2ce7

**Comment by da2ce7** [comment]
I guess the only counter-proposal for this one is to merge the https://github.com/torrust/torrust-index-frontend/pull/73 and publish a crate.

**Comment by josecelano** [comment]
Alternatives:

- https://vitepress.dev/
